ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-133] Serialize manage no income attributes

**N.B. Merge https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/pull/559 first**
**N.B. Merge https://github.com/ministryofjustice/laa-criminal-legal-aid-schemas/pull/60 and update schema version first**

## Description of change
Add manage_without_income and manage_other_details attributes to income_details

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAPP-133

## How to manually test the feature

1. Point to new schema commit github: 'ministryofjustice/laa-criminal-legal-aid-schemas', ref: 'b1d93da06b376a132af6f5f783a4eb989c7de6d8' in Gemfile and bundle
2. Create a new application, fill all details but DO NOT SUBMIT
3. Go to /steps/income/what_is_clients_employment_status
4. Select 'My client is not working', Select 'Yes'
5. Click Save and continue
6. Select 'Yes', Input 01/10/2023
7. Click Save and continue
8. Select 'Other', input some text
9. Click Save and come back later
11. Go to /steps/income/clients_income_before_tax
12. Select 'Yes'
13. Click Save and continue
14. Reopen that application
15. Click Confirm declaration and submit application
16. Click Save and submit
17. Make sure it succeeds
